### ISSUE TITLE
Include package sizes of final bundle

### ISSUE THREAD

**Issue by ayuhito** [privatenumber/pkgroll#12]
Similar to other bundlers:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/38220115/176196413-34152ec1-8c2b-4da0-9681-e9e75cf7dc50.png">

**Comment by privatenumber** [comment]
I contemplated this at one point, but I usually want to know the size of the entire package (not just the distribution files) so I run `npx pkg-size` to check.

I guess I wouldn't mind supporting this though!

**Comment by ayuhito** [comment]
Food for thought, but I think one really interesting metric would be measuring individual exports so you can get it's tree-shakened size for each export.

I don't think that's an easy thing to do though compared to just providing the size of the output files.

Either way, even integrating pkg-size to reduce the number of input commands would be ideal per build.

**Comment by privatenumber** [comment]
Yeah that would be cool but probably a cherry on top. I actually wanted to write a module that helps calculate that.

We probably shouldn't integrate `pkg-size` to a bundler, because `pkg-size` is concerned about the entire package size (including `README`, `LICENSE`, `package.json`, etc) whereas the bundler is only concerned about the files it produced.

**Comment by ayuhito** [comment]
Just for future reference, but I think this project does implement all the treeshaking logic discussed before using esbuild: https://github.com/okikio/bundle
